pkg/passphrase/passphrase.go

```diff
@@ -18,6 +18,16 @@ import (
 // decide how many chances to give to a human, for example.
 type Retriever func(keyName, alias string, createNew bool, attempts int) (passphrase string, giveup bool, err error)
 
+const (
+	tufRootAlias                = "root"
+	tufTargetsAlias             = "targets"
+	tufSnapshotAlias            = "snapshot"
+	tufRootKeyGenerationWarning = `You are about to create a new root signing key passphrase. This passphrase will be used to protect
+the most sensitive key in your signing system. Please choose a long, complex passphrase and be careful
+to keep it secure and backed up. It is highly recommended that you use a password manager to both generate it
+and keep it safe. There will be no way to recover this key.`
```

Comment:
I'd suggest wrapping this at a shorter line length. It will look weird on a standard (80 column) terminal.